packages/client/src/modals/Transaction/BannerInfo.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useContext, useState } from "react";
+import { Web3Context } from "contexts/Web3";
+import { ACTION_TYPES } from "constants/enums";
+import Svg from "library/Svg";
+import { formatAddress, getTokenMeta, parseIdentifier, getNameByAddress } from "utils";
+
+const BannerInfo = ({ className = "", actionData = {}, contacts = {}, signers = [] }) => {

```

Comment:
Didn't see any place passing the signers property to BannerInfo, is it being used?

Reply:
ah yeah it was missing being passed down from `Details.js`, added it!